02-technical-guides/aggregate-provider/main.ipynb

```diff
@@ -346,7 +346,7 @@
        "</ul>\n"
```

Comment:
<p>Numbers seem to have changed, is this expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Verified with <code>0.8.10</code>, and these numbers are expected to change, most likely due to the <code>quantile</code> function. Resolved.  </p>